src/Lithnet.GoogleApps.MA/ApiInterfaces/ApiInterfaceUserSendAs.cs

```diff
@@ -167,9 +167,12 @@ private AttributeChange ApplySendAsChanges(CSEntryChange csentry)
                     SendAs sendAs = new SendAs
                     {
                         DisplayName = address.DisplayName,
-                        SendAsEmail = address.Address
+                        SendAsEmail = address.Address,
                     };
 
+                    if (this.config.MakeNewSendAsAddressesDefault)
+                        sendAs.IsDefault = true;
```

Comment:
Sticking with existing style conventions we need to brace the conditional expression

Reply:
Done, new Commit coming.